### DISCUSSION TITLE
During a test we collect jaeger trace, attach it. Maybe provide some UI to see it?

### DISCUSSION THREAD

**Discussion by neopaf** [allure-framework/allure2#2565] [Ideas & Feedback]
https://www.jaegertracing.io

.js format trace.... we'd like it to be viewable from Allure report without extra actions.

**Comment by neopaf** [comment]
Some random UI screenshots:
![44633315-cc526180-a957-11e8-8b9f-e4e51961d56b](https://github.com/allure-framework/allure2/assets/4362564/9564413f-9e87-4f4f-a42b-6f52c59c77a6)

**Reply by baev** [reply]
We're researching ways to support different tracing formats, e.g., supporting HAR files. I will look into this one as well, thanks

**Reply by neopaf** [reply]
Dmitry, thanks!

Maybe you'd consider Google Tracing format as well.

Jaeger is [convertable](https://stackoverflow.com/questions/70337528/how-to-view-a-jaeger-trace-in-chrome-flame-graph) to Google format
